### PR TITLE
Add CI environment so it can be fabulous

### PR DESCRIPTION
**Pull request by benthorner**
https://trello.com/c/0Gi6yGMQ/603-fix-metrics-not-working-on-all-instances-in-the-ci-environment

Previously we used a _set_gateway helper to assign the appropriate
jumpbox for the environment; this wasn't compatible with the domain
we use for the CI jumpbox, so this inlines the one line of this
fucntion into each environment function - it's also shorter!

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -153,7 +135,15 @@ def aws_production(stackname=None):
     """Select GOV.UK AWS Production  environment"""
     env['environment'] = 'production'
     env['aws_migration'] = True
-    _set_gateway("{}.production.govuk.digital".format(stackname))
+    env.gateway = 'jumpbox.blue.production.govuk.digital'
+
+
+@task
+def ci(stackname=None):
+    """Select GOV.UK AWS Production  environment"""
```
Comment (thomasleese) [inline fabfile.py]: We need to change the doc string here.

Comment (benthorner) [reply fabfile.py]: Oops - corrected.

File: fabfile.py [hunk 2]
```diff
@@ -103,35 +94,26 @@ def help(name=""):
...
 @task
 def integration(stackname=None):
-    if not stackname:
-        stackname = 'blue'
-
     """Select integration environment"""
     env['environment'] = 'integration'
     env['aws_migration'] = True
-    _set_gateway("{}.integration.govuk.digital".format(stackname))
+    env.gateway = 'jumpbox.blue.integration.govuk.digital'
```
Comment (thomasleese) [inline fabfile.py]: If we're no longer using the `stackname` parameter, I think we should remove it from the argument list to avoid any confusion.

Comment (benthorner) [reply fabfile.py]: Oh yeah - thanks for spotting. I should have been paying more attention :-|.

**Pull request merged**